Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -122,6 +123,36 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
             this._fontVariationsGroup.rows = [emptyRow];
         }
+
+        // Custom Font Properties
+
+        // Display message when we have no custom fonts.
+        if (!this.nodeStyles.customFonts.length) {
+            this._customFontsRow.showEmptyMessage();
+            this._customFontsRow.dataGrid = null;
+            return;
+        }
+
+        // We can have no dataGrid if we navigate to a page
+        // with no custom fonts. This will trigger it to force
+        // rebuild the table.
+        if (!this._customFontsRow.dataGrid)
+            this._customFontsRow.dataGrid = this._dataGrid;
```

Comment:
rather than keeping the `_allFontsDataGrid` as a member variable, i think instead we should `return` it from `_createAllFontsDataGrid` and then have that be called here
```js
this._customFontsRow.dataGrid ||= this._createAllFontsDataGrid();
this._customFontsRow.dataGrid.removeChildren();
...
```